Quiz/index.html

```diff
@@ -4,33 +4,50 @@
 		<meta charset="UTF-8" />
 		<title>Quiz</title>
 		<link rel="stylesheet" href="./style.css" />
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/pretty-checkbox@3.0/dist/pretty-checkbox.min.css"
+		/>
 	</head>
 	<body>
-		<div>
+		<div class="wrapper">
```

Comment:
![image](https://user-images.githubusercontent.com/67651737/185181333-274a04f7-c4e9-4352-8c3c-79b3210957a5.png)

Not responsive height and width